README.md

```diff
@@ -6,17 +6,45 @@
 
 ## What is Rust?
 
-At a high level Rust is a language thats goal is to allow programmers to get systems-level performance without devoting the time to learn about manual memory managment. It also comes out of the box with a set of tools to steer you towards writing better, more optomized code.
+At a high level Rust is a language thats goal is to allow programmers to get systems-level performance without devoting the time to learn about manual memory management. It also comes out of the box with a set of tools to steer you towards writing better, more optimized code.
```

Comment:
Nit: thats should be that’s, but I generally prefer whose for this kind of usage, or the much more stilted, “language, the goal of which is...”